### PR TITLE
Add artifacts to circle

### PR DESCRIPTION
**Pull request by Kubuxu**
License: MIT

### CODE REVIEW

**Comment by whyrusleeping** [comment]
@Kubuxu can you add Mac builds and also provide some instructions in the readme on how to download these?

File: .circleci/config.yml [hunk 1]
```diff
@@ -109,6 +109,20 @@ jobs:
...
+    executor: golang
+    steps:
+      - install-deps
+      - prepare
+      - go/mod-download
+      - run:
+          command: make -j2 build
+      - store_artifacts:
+          path: lotus
+          destination: lotus
```
Comment (magik6k) [inline .circleci/config.yml]: Having some way to tell what the platform is would be nice, something like:
```suggestion
          destination: lotus.amd64.linux
```

**Comment by Kubuxu** [comment]
This is WIP, Circle artefacts have bad UX. We will probably go with nightly tagged builds.

**Comment by whyrusleeping** [comment]
@Kubuxu should we close this then? or is there stuff in here that we still want?

**Pull request closed without merging**